src/Java.Interop/Java.Interop/JniRuntime.JniMarshalMemberBuilder.cs

```diff
@@ -35,6 +35,7 @@ partial class CreationOptions {
 			} catch (Exception) {
 				return;
 			}
+			System.Diagnostics.Debug.WriteLine ($"Java.Interop.Export assembly loaded ({jie.FullName})");
```

Comment:
I think in this case, instead of logging that the assembly was loaded, we should instead log the exception on failure.

Reply:
OK, updated.